nodebuilder/node.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/celestiaorg/celestia-node/nodebuilder/state"
 )
 
-var Timeout = time.Second * 30
+var Timeout = time.Minute * 2
```

Comment:
It's not sustainable to increase all the time, so we should consider removing timeout for BN/LNs or have diff timeouts for node types. Light node should definitely start quick and within 15 secs